openvdb/openvdb/Platform.h

```diff
@@ -229,4 +229,54 @@
     #define OPENVDB_HOUDINI_API OPENVDB_IMPORT
 #endif
 
+#if defined(__ICC)
```

Comment:
Tempted to say we strip all of this out - we don't build or test for ICC so it's hard to know if any of this is still relevant. They're just suppressing warnings after all.

Reply:
Some of these defines are still in use so no harm leaving them in for now